src/hooks/useLoadClientScript.ts

```diff
@@ -1,31 +1,30 @@
-import { useState, useMemo, useEffect } from "react"
+import { useState, useEffect } from "react"
 import { isNostoLoaded } from "../components/helpers"
 import type { NostoClient } from "../types"
 import type { NostoProviderProps } from "../components/NostoProvider"
 
 type NostoScriptProps = Pick<NostoProviderProps, "account" | "host" | "shopifyMarkets" | "loadScript">
 
 export function useLoadClientScript(props: NostoScriptProps) {
-  const { host, account, shopifyMarkets, loadScript = true } = props
-  const [clientScriptLoadedState, setClientScriptLoadedState] = useState(false)
-  const clientScriptLoaded = useMemo(() => clientScriptLoadedState, [clientScriptLoadedState])
```

Comment:
couldn't figure out the purpose of this wrapping
useMemo wrapping didn't seem to have any impact here

Reply:
`clientScriptLoadedState` is already memoized by `useState` internally, so it's not needed here, yes.